### ISSUE TITLE
[CI] LegacyGeoShapeFieldMapperTests testMinimalIsInvalidInRoutingPath failing

### ISSUE THREAD

**Issue by fcofdez** [elastic/elasticsearch#81415]
**Build scan:**
https://gradle-enterprise.elastic.co/s/l3web4uikmzda/tests/:modules:legacy-geo:test/org.elasticsearch.legacygeo.mapper.LegacyGeoShapeFieldMapperTests/testMinimalIsInvalidInRoutingPath

**Reproduction line:**
`./gradlew ':modules:legacy-geo:test' --tests "org.elasticsearch.legacygeo.mapper.LegacyGeoShapeFieldMapperTests.testMinimalIsInvalidInRoutingPath" -Dtests.seed=9D9C10E69693AB84 -Dbuild.snapshot=false -Dtests.jvm.argline="-Dbuild.snapshot=false" -Dtests.locale=fi -Dtests.timezone=America/Bahia_Banderas -Druntime.java=17`

**Applicable branches:**
master

**Reproduces locally?:**
Didn't try

**Failure history:**
https://gradle-enterprise.elastic.co/scans/tests?tests.container=org.elasticsearch.legacygeo.mapper.LegacyGeoShapeFieldMapperTests&tests.test=testMinimalIsInvalidInRoutingPath

**Failure excerpt:**
```
java.lang.IllegalArgumentException: setting index.time_series.start_time has not been registered

  at __randomizedtesting.SeedInfo.seed([9D9C10E69693AB84:C8CED0B44AAE8F09]:0)
  at org.elasticsearch.common.settings.AbstractScopedSettings.get(AbstractScopedSettings.java:740)
  at org.elasticsearch.index.TimestampBounds.<init>(TimestampBounds.java:22)
  at org.elasticsearch.index.IndexSettings.<init>(IndexSettings.java:701)
  at org.elasticsearch.index.IndexSettings.<init>(IndexSettings.java:680)
  at org.elasticsearch.index.mapper.MapperServiceTestCase.createIndexSettings(MapperServiceTestCase.java:215)
  at org.elasticsearch.index.mapper.MapperTestCase.testMinimalIsInvalidInRoutingPath(MapperTestCase.java:750)
  at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(NativeMethodAccessorImpl.java:-2)
  at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
  at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  at java.lang.reflect.Method.invoke(Method.java:568)
  at com.carrotsearch.randomizedtesting.RandomizedRunner.invoke(RandomizedRunner.java:1758)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$8.evaluate(RandomizedRunner.java:946)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$9.evaluate(RandomizedRunner.java:982)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$10.evaluate(RandomizedRunner.java:996)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at org.apache.lucene.util.TestRuleSetupTeardownChained$1.evaluate(TestRuleSetupTeardownChained.java:44)
  at org.apache.lucene.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
  at org.apache.lucene.util.TestRuleThreadAndTestName$1.evaluate(TestRuleThreadAndTestName.java:45)
  at org.apache.lucene.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:60)
  at org.apache.lucene.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:44)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:375)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl.forkTimeoutingTask(ThreadLeakControl.java:824)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl$3.evaluate(ThreadLeakControl.java:475)
  at com.carrotsearch.randomizedtesting.RandomizedRunner.runSingleTest(RandomizedRunner.java:955)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$5.evaluate(RandomizedRunner.java:840)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$6.evaluate(RandomizedRunner.java:891)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$7.evaluate(RandomizedRunner.java:902)
  at org.apache.lucene.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at org.apache.lucene.util.TestRuleStoreClassName$1.evaluate(TestRuleStoreClassName.java:38)
  at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)
  at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at org.apache.lucene.util.TestRuleAssertionsRequired$1.evaluate(TestRuleAssertionsRequired.java:53)
  at org.apache.lucene.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
  at org.apache.lucene.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:44)
  at org.apache.lucene.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:60)
  at org.apache.lucene.util.TestRuleIgnoreTestSuites$1.evaluate(TestRuleIgnoreTestSuites.java:47)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:375)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl.lambda$forkTimeoutingTask$0(ThreadLeakControl.java:831)
  at java.lang.Thread.run(Thread.java:833)

```

**Comment by dimitris-athanasiou** [comment]
Another failure. Couldn't reproduce locally.

https://gradle-enterprise.elastic.co/s/4uzgljaghk52a/tests/:modules:legacy-geo:test/org.elasticsearch.legacygeo.mapper.LegacyGeoShapeFieldMapperTests/testMinimalIsInvalidInRoutingPath?top-execution=1

**Comment by dimitris-athanasiou** [comment]
This seems related to #81146. Perhaps @nik9000 could take a look as was the reviewer of that PR?

**Comment by dimitris-athanasiou** [comment]
We got another failure of this in 7.16: https://gradle-enterprise.elastic.co/s/yafgnmrinitmq.

Should we backport the fix #81494?
